CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Release Candidate
 
+## Refactoring
+
+- Add a commit abbreviation to Version in UserInterfaceCard (INDIGO Sprint 220722, [!334](https://github.com/TeskaLabs/asab-webui/pull/334))
+
+## v22.29
```

Comment:
@petrKavulok This is not necessary in github repo, I did not manage to create a release here. When I will, I'll inform you. Sorry for messing things around. Btw. there is no `v22.29` tag anywhere version